crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def check_unclaimed_disks
+    unclamed_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
+
+      unless disk =~ /^\/dev\/disk\//
+        unclamed_disks.push(device_name) unless node[:block_device][path].nil?
+        next
+      end
+
+      devices = node[:block_device].map do |device, _claim|
```

Comment:
Nearly sure that there's no claim-related bits in `node[:block_device]`, so at least the variables here are wrong. But more importantly, wouldn't it make sense to create the list of devices paths outside of the `each` loop (without checking for `path`), and then just do a `include?` test here?